silver/tests/unit/test_payments_util.py

```diff
@@ -78,16 +83,21 @@ def test_get_transaction_from_expired_token(self):
         mocked_view.has_calls([call(None, transaction, True), call()])
 
     @override_settings(PAYMENT_METHOD_SECRET='a')
-    @override_settings(SILVER_PAYMENT_TOKEN_EXPIRATION=timedelta(minutes=1))
+    @override_settings(SILVER_PAYMENT_TOKEN_EXPIRATION=timedelta(seconds=60))
     def test_get_jwt_token(self):
-        uuid = UUID('6fa459ea-ee8a-3ca4-894e-db77e160355e', version=4)
+        uuid = UUID('6fa459ea-ee8a-4ca4-894e-db77e160355e', version=4)
         transaction = TransactionFactory(uuid=uuid)
-
-        expected_token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0cmFuc2FjdG' \
-                         'lvbiI6IjZmYTQ1OWVhLWVlOGEtNGNhNC04OTRlLWRiNzdlMTYwM' \
-                         'zU1ZSIsImV4cCI6MTQ5Nzk2NTY0MH0.-bpx5A3DfSe3-HO6aH_g' \
-                         'lS8adcCxUn8lSK1-RPxohhI'
+        if six.PY3:
+            expected_token = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ0cmFuc2FjdG' \
+                             'lvbiI6IjZmYTQ1OWVhLWVlOGEtNGNhNC04OTRlLWRiNzdlMTYwM' \
+                             'zU1ZSIsImV4cCI6MTQ5Nzk2NTY0MH0.NuqwXlHt27nRxg5W2hHo' \
+                             'P5ugnLmw-7QYBumO7lRa1i0'
+        else:
+            expected_token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ0cmFuc2FjdG' \
```

Comment:
I don't understand how this token remained the same even though you modified the `uuid` above. :confused:

Reply:
Yeah I didn't quite understand that either. I just figured it was a typo.

Reply:
The collision is probably due to the weak secret that is being used
```python
@override_settings(PAYMENT_METHOD_SECRET='a')
```

Reply:
`timedelta(minutes=1)` is equivalent to `timedelta(seconds=60)`. It's probably the just the weak secret value. We can address that some other time...